src/helpers/enrich-dataset.ts

```diff
@@ -55,12 +54,35 @@ export function enrichDataset(dataset: HubDataset, hubsite: HubSite) {
         additionalFields.accessUrlWMS = ogcUrl(dataset.url, 'WMS');
     }
 
-    return {
+    return hubDatasetToFeature({
         ...dataset,
         ...additionalFields
-    };
+    });
 };
 
+function hubDatasetToFeature(hubDataset: HubDataset) {
+    const { type, rings } = hubDataset?.boundary?.geometry ?? {};
+
+    return {
+        type: 'Feature',
+        geometry: {
+            type: type && capitalize(type),
+            coordinates: rings
+        },
+        properties: _.omit(hubDataset, ['boundary'])
+    };
+}
+
+function capitalize(word) {
+    return word
+        .split('')
+        .map((letter, index) =>
+            index ? letter.toLowerCase() : letter.toUpperCase(),
+        )
+        .join('');
+}
```

Comment:
Maybe more efficient to do a lookup with an object?

`{ polygon: 'Polygon' }`

We should only ever have polygon in the index, but if you want to future proof: https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-shape.html#input-structure

I am a little surprise the search API doesn't transform this for us before we get here.  I would ask Tom Wayson if the current format delivered by the search API should be altered.

Reply:
Used look up for mapping elastic type to GeoJSON geometry type.